### DISCUSSION TITLE
Windows 10 21H2 search highlights

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#2643] [Show and tell]
I think that option would be good and work on taking the code from the windows 10 search box and making it a 1:1 replica of that bar.....

this option can be toggled on if you want and toggled off if you don't want it. this with the windows 10 search menu enabled by default and with the option to enable the windows 11 one... (the windows 10 search menu probably needs to be implemented) plus with the unasked highlight the-
![Screenshot 2023-08-02 131957](https://github.com/valinet/ExplorerPatcher/assets/140480110/0750eb28-987f-46f3-a0ef-8898052e0430)
(this is a concept of how it might look like, yes i put the windows 10 taskbar on top of the image yes...)
![show-search-highlights](https://github.com/valinet/ExplorerPatcher/assets/140480110/5a67d639-1239-4c90-9292-5886dd6d3bf4)

**Comment by pyrates999** [comment]
Windows 10 21h2 is not tested on by EP.

**Comment by jayfreddy1994** [comment]
i think, EP is much better with current development for now. it's made for the sake of windows 11
